### DISCUSSION TITLE
Error on New Environment

### DISCUSSION THREAD

**Discussion by SilvaFernando** [gravitational/teleport#45591] [Q&A] [answered]
Expected behavior:
Authenticate with TOTP 

Current behavior:
When I tried logging on to TEleport with my credentials and my token, not work, I've tried creating a new user:
![image](https://github.com/user-attachments/assets/336b73bf-d277-4bee-b88b-19fdf6131307)
Follow Logs:
[teleport_logs.txt](https://github.com/user-attachments/files/16661747/teleport_logs.txt)

My Environment:

- VPS on hetzner - on EUA Location
- Client-side in Brazil

I've tried to adjust a date/time on my server side and in my client-side/

Bug details:
- Teleport version: 16.1.4
- Recreation steps: Create a user
- Debug logs:

**Comment by SilvaFernando** [comment]
Hi team.
If I'm doing something wrong let me know.

To install this Teleport this doc was used: https://goteleport.com/docs/installation/#docker

**Comment by webvictim** [comment]
Generally the advice would be to ensure that the time on both server and client is synced to a network time server, for example using NTP/chrony. It sounds like you're saying you've done that, though? It's worth making 100% sure that both are in sync.

Also, what app are you using to scan the TOTP code? Could you try a different app in case that's the issue?

**Comment by SilvaFernando** [comment]
Hi @webvictim/

I checked NTP and I didn't find any error:
![image](https://github.com/user-attachments/assets/cbf89e78-bc49-451c-ae09-2ce0b3c5beb9)

I'm using Google Authenticator, but I tried with Microsoft authenticator too.

**Reply by webvictim** [reply, marked as answer]
I think that's actually indicating the issue - the `System clock synchronized: no` message means that the clock is out of sync.

Here is some output from a server with correct time sync:
```
gus@hades:~$ timedatectl
               Local time: Thu 2024-08-22 12:33:29 ADT
           Universal time: Thu 2024-08-22 15:33:29 UTC
                 RTC time: Thu 2024-08-22 15:33:29
                Time zone: America/Halifax (ADT, -0300)
System clock synchronized: yes
              NTP service: active
          RTC in local TZ: no
```

Take a look at this discussion for some information on how to fix it: https://askubuntu.com/questions/1058593/how-to-sync-the-time-to-network-with-timedatectl-on-ubuntu-18-04

**Reply by SilvaFernando** [reply]
I adjusted this and worked!!!!

Thanks
